SPECS/initramfs/initramfs.spec

```diff
@@ -88,15 +88,17 @@ echo "initramfs (re)generation" %* >&2
 cat > /dev/null \
 if [ -f %{_localstatedir}/lib/rpm-state/initramfs/regenerate ]; then \
     echo "(re)generate initramfs for all kernels," %* >&2 \
-    mkinitrd -q \
+    mkinitrd --preload megaraid_sas --preload megaraid -q \
```

Comment:
I have some concerns about forcing these drivers into every image's initramfs. Would it be acceptable to move this to the dracut conf files? You can either add that line in a post install script + invoke mkinird again, or if you want something a bit more stable maybe we could consider adding a package? (I had to add drivers for my verity module here:  https://github.com/microsoft/CBL-Mariner/blob/2.0/SPECS/verity-read-only-root/verity.conf . There are both normal driver directives, and a driver_force option.